packages/mapviewer/tests/cypress/tests-e2e/featureSelection.cy.js

```diff
@@ -344,6 +344,23 @@ describe('Testing the feature selection', () => {
             })
         }
 
+        // This function simulates a click on the map at the specified location {x, y}
+        // The ctrlKey parameter allows for simulating a click with the CTRL key pressed.
+        function clickOnMap(location, ctrlKey = false) {
+            cy.get('@olMap').realMouseDown({
+                x: location.x,
+                y: location.y,
+                position: 'topLeft',
+                ctrlKey: ctrlKey,
+            })
+            cy.get('@olMap').realMouseUp({
+                x: location.x,
+                y: location.y,
+                position: 'topLeft',
+                ctrlKey: ctrlKey,
+            })
```

Comment:
Can you get away of `cypress-real-events` and use a standard `.click` call?
I know the OL map can be a heal to deal with, so maybe not 🙃